frontend/io-webapp/src/session/Creator.js

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+
+const Creator = (props) => {
+  const [abcd, setAbcd] = React.useState(false);
+  const [question, setQuestion] = React.useState("");
+  const [answer, setAnswer] = React.useState("");
+  const [answers, setAnswers] = React.useState(["", "", "", ""]);
+
+  const submitHandler = (e) => {
+    /* TODO */
+    e.preventDefault();
+    if (question !== "") {
+      const newQuestion = { question };
+      if (answer !== "") {
+        newQuestion.answer = { answer };
+        console.log(newQuestion);
+      } else if (
+        answers.length === 4 &&
+        answers[0] !== "" &&
+        answers[1] !== "" &&
+        answers[2] !== "" &&
+        answers[3] !== ""
+      ) {
+        newQuestion.answers = { answers };
+        console.log(newQuestion);
+      }
+    }
+    setQuestion("");
+    setAnswer("");
+    setAnswers(["", "", "", ""]);
+  };
+
+  return (
+    <div className="question" style={{ width: props.width }}>
+      <form onSubmit={submitHandler}>
+        <div>
+          <label htmlFor="questionText">Question: </label>
+          <input
+            type="text"
+            id="questionText"
+            name="questionText"
+            value={question}
+            onChange={(e) => setQuestion(e.target.value)}
+          ></input>
+        </div>
+        <button type="button" onClick={() => setAbcd(!abcd)}>
+          {abcd ? "ABCD" : "Open"}
+        </button>
+        <div>
+          {abcd ? (
+            <>
+              <label htmlFor="aText">A: </label>
+              <input
+                type="text"
+                id="aText"
+                name="aText"
+                value={answers[0]}
+                onChange={(e) =>
+                  setAnswers([
+                    e.target.value,
+                    answers[1],
+                    answers[2],
+                    answers[3],
+                  ])
+                }
+              ></input>
```

Comment:
O ile dobrze widzę to jest skopiowane 4 razy. Najłatwiej by było tego pewnie uniknąć nadbudowywując `setAnswers` jeszcze jakąś funkcją (powiedzmy taką, która będzie przyjmować nową odpowiedź i [0, 1, 2, 3]), tak, żeby dla każdej odpowiedzi móc ją wołać tak samo